cmd/connect.go

```diff
@@ -53,6 +53,9 @@ Then paste the token`,
 	Run: func(cmd *cobra.Command, args []string) {
 		flag.Parse()
 
+		// Assert AWS Creds
+		handler.AssertCredentials()
+
```

Comment:
Moved to the `Run` function instead of `init()`, as `init()` runs every time, even when this specific command isn't called, so we were calling `handler.AssertCredentials()` twice now (and more if we add more commands)